Rakefile

```diff
@@ -1,5 +1,4 @@
-require "bundler/gem_tasks"
```

Comment:
I use the `release` script every time I have to release a new version, so I would like to keep this dependency.
```sh
rake build            # Build urn-XXX.gem into the pkg directory
rake clean            # Remove any temporary products
rake clobber          # Remove any generated files
rake install          # Build and install urn-XXX.gem into system gems
rake install:local    # Build and install urn-XXX.gem into system gems without network access
rake release[remote]  # Create tag vXXX and build and push urn-XXX.gem to Rubygems
```